Dockerfile

```diff
@@ -12,11 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-ARG BASE_IMAGE=nvcr.io/nvidia/tritonserver:23.03-py3
-ARG TRITONSDK_BASE_IMAGE=nvcr.io/nvidia/tritonserver:23.03-py3-sdk
+ARG BASE_IMAGE=nvcr.io/nvidia/tritonserver:23.04-py3
+ARG TRITONSDK_BASE_IMAGE=nvcr.io/nvidia/tritonserver:23.04-py3-sdk
 
-ARG MODEL_ANALYZER_VERSION=1.27.0dev
-ARG MODEL_ANALYZER_CONTAINER_VERSION=23.04dev
+ARG MODEL_ANALYZER_VERSION=1.28.0dev
```

Comment:
I don't think these are supposed to be updated now? Those are updated after freeze.

Reply:
I see. We missed this last time so updating it now. This is fine